src/virtual-select.js

```diff
@@ -661,7 +661,17 @@ export class VirtualSelect {
       valueTooltip.push(`<span class="vscomp-value-tag">+ ${moreSelectedOptions} more...</span>`);
     }
 
-    this.$valueText.innerHTML = valueText.join(', ') || this.placeholder;
+    const aggregateValueText = valueText.join(', ');
+    if (aggregateValueText === '') {
+      this.$valueText.innerHTML = this.placeholder;
+    } else {
+      this.$valueText.innerHTML = aggregateValueText;
+      if (this.isEllipsisActive(this.$valueText)) {
+        /** replace comma delimitted list of selections with shorter text indicating selection count */
+        this.$valueText.innerHTML = `${valueText.length} option${valueText.length === 1 ? '' : 's'} selected`
```

Comment:
Use `this.selectedValues.length` instead of `valueText.length`

```js
let selectedLength = this.selectedValues.length;
this.$valueText.innerHTML = `${selectedLength} option${selectedLength === 1 ? '' : 's'} selected`;
```

Reply:
Will, do thanks!